### PR TITLE
Remove `minimum_utxo_value` field from REST API.

### PR DESCRIPTION
**Pull request by jonathanknowles**
## Issue Number

ADP-2250

## Summary

This PR removes the `minimum_utxo_value` field from the REST API.

Future PRs will improve our API documentation relating to the treatment of minimum UTxO values by other endpoints.

### CODE REVIEW

**Comment by jonathanknowles** [comment]
bors try

**Review by paweljakubas — APPROVED** [review]
lgtm

**Review by Anviking — APPROVED** [review]

File: specifications/api/swagger.yaml [hunk 1]
```diff
@@ -2228,69 +2227,6 @@ components:
...
-            the minimum ada quantity.
-
-            In the Babbage era, the `coinsPerUTxOWord` protocol parameter was
-            replaced by the `coinsPerUTxOByte` protocol parameter. In this era,
-            the minimum ada quantity for an output is determined by multiplying
-            the `coinsPerUTxOByte` parameter by the length (in bytes) of the
-            serialised representation of the output, which is dependent on the
-            length of the address (among other factors).  Therefore, in this
-            era, specifying a longer address will require an increase in the
-            minimum ada quantity.
```
Comment (jonathanknowles) [inline specifications/api/swagger.yaml:2292]: In a subsequent PR, I will re-add parts of this description to the descriptions of other endpoints. In particular, I aim to ensure that all endpoints involving transaction construction have suitable documentation of the following behaviour:
- If the user specifies a transaction output `out` without any ada (or an ada quantity of zero), the wallet will automatically assign a minimal quantity of ada that satisfies the ledger rules.
- If the user specifies a transaction output `out` with an ada quantity `q`, the wallet will verify that `q` is large enough to satisfy the ledger minimum UTxO rule, and if `q` is too low, return an error with a revised quantity `r. r > q` that does satisfy the ledger minimum UTxO rule. The onus is then on the user to resubmit their request with the revised quantity `r` (or some value that is greater than `r`).

**Comment by jonathanknowles** [comment]
Will merge manually, as:
- [x] I have run all test suites locally with no failures.
- [x] I have successfully run at least one `bors try` (with no failures).
- [x] The most recent `bors r+` attempt failed with just one error that is:
   - caused by a known flaky test.
   - unrelated to this PR.
- [x] After restarting the failed job, it failed again, but with the same flaky test that is unrelated to this PR.

**Pull request merged**